src/util/isGif.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function(url){
+    
+  const toMatch = url.match('data:image/(.*);');
```

Comment:
It seems this function is glitching. I think URL here is undefined in some cases. Please check that.

Reply:
:thinking: ok, on it!

Reply:
Yes you were right, I fixed it, now tests are running fine, Thanks!!!